README.markdown

```diff
@@ -309,62 +310,116 @@ Manages a setting within an INI file.
 
 ##### `ensure`
 
-Determines whether the specified setting should exist. Valid options: 'present' and 'absent'. Default value: 'present'.
+Determines whether the specified setting should exist. 
+
+Valid options: 'present' and 'absent'. 
+
+Default value: 'present'.
 
 ##### `key_val_separator`
 
-*Optional.* Specifies a string to use between each setting name and value (e.g., to determine whether the separator includes whitespace). Valid options: a string. Default value: ' = '.
+*Optional.* 
```

Comment:
Doesn't need to be specified as *Optional*. Instead, under Parameters, add a sentence that's something like "all parameters are optional unless specified as required", and get rid of the all the *Optional* notations throughout the README.